pkg/nvkind/node.go

```diff
@@ -50,7 +50,7 @@ func (n *Node) InstallContainerToolkit() error {
 
 func (n *Node) ConfigureContainerRuntime() error {
 	err := n.runScript(`
-	    nvidia-ctk runtime configure --runtime=containerd --set-as-default
+	    nvidia-ctk runtime configure --runtime=containerd --set-as-default --config-source=command
```

Comment:
This is fine. As a follow-up, I think we should remove the `--set-as-default` option -- or make it opt-in. The purpose is to allow for testing scenarios that our customers would use, and not all of them would set the `nvidia` runtime as the default.

Reply:
done